roles/cfg_openwrt/tasks/imagebuilder.yml

```diff
@@ -105,6 +105,7 @@
     path: "{{ configs_dir }}/etc/apk/repositories.d/falter.list"
     line: "{{ feed }}"
     create: true
+    mode: "0644"
```

Comment:
`mode` seems to be an actual functional check. This I find acceptable to result in a check fail blocking a PR from being merged.

But all the other stuff is just formatting and shouldn't be part of a "lint" task, I think.